server/src/server.ts

```diff
@@ -1936,7 +1920,7 @@ function initializePolisHelpers() {
     // Argument of type '{ path: string; headers?: { host: string; } | undefined; }' is not assignable to parameter of type 'Req'.
     //  Property 'cookies' is missing in type '{ path: string; headers?: { host: string; } | undefined; }' but required in type 'Req'.ts(2345)
     // @ts-ignore
-    cookies.setCookieTestCookie(req, res, shouldSetCookieOnPolisDomain(req));
```

Comment:
interesting this one, I wonder how it worked beforehand as I don't think I moved this function 🤔

Reply:
¯\_(ツ)_/¯

Its probably the direct import of `shouldSetCookieOnPolisDomain` that was changed at the top - there is no reason to call `setCookieTestCookie` and `shouldSetCookieOnPolisDomain` differently so that is a reasonable refactor but the code here just wasn't updated inline